src/applyMark.ts

```diff
@@ -31,6 +31,17 @@ function markApplies(
   return false;
 }
 
+function hasMark(marks, markType) {
+  let has = false;
+  marks.forEach(function (mark) {
+    if (mark.type.name === markType) {
+      has = true;
```

Comment:
No any on parameters.

If marks is an Array, this can be simplified to `return marks.some(x => x.mark.type === markType)`

Otherwise, remove the has variable. Return true when found, and return false at the end if it wasn't found.